### PR TITLE
Update document

### PR DESCRIPTION
**Pull request by hrchu**

### CODE REVIEW

File: confluent_kafka/src/Consumer.c [hunk 1]
```diff
@@ -1236,7 +1236,8 @@ static PyMethodDef Consumer_methods[] = {
           "  :param bool cached: Instead of querying the broker used cached information. "
           "Cached values: The low offset is updated periodically (if statistics.interval.ms is set) while "
           "the high offset is updated on each message fetched from the broker for this partition.\n"
-          "  :returns: Tuple of (low,high) on success or None on timeout.\n"
+          "  :returns: Tuple of (low,high) on success or None on timeout."
```
Comment (edenhill) [inline confluent_kafka/src/Consumer.c]: Add a space after the period: `...timeout. "`

File: confluent_kafka/src/Consumer.c [hunk 2]
```diff
@@ -1236,7 +1236,8 @@ static PyMethodDef Consumer_methods[] = {
...
+	  "The high offset is the offset of the last message + 1.\n"
```
Comment (edenhill) [inline confluent_kafka/src/Consumer.c]: Use 8 space as indent, not tab.

**Comment by matt2000** [comment]
For my 2c, this is a useful clarification, and it looks like the author made the requested changes.

**Comment by edenhill** [comment]
[clabot:check]

**Comment by edenhill** [comment]
Thank you!

**Pull request merged**